File: model/src/main/java/com/sealinkin/cost/model/Cost_AccountSetModel.java
package com.sealinkin.cost.model;

import com.sealinkin.cost.po.Cost_AccountSet;

public class Cost_AccountSetModel extends Cost_AccountSet {

	private static final long serialVersionUID = 1L;
	private String enterpriseNo;
	private String accountNo;
	
	public String getEnterpriseNo() {
		return enterpriseNo;
	}
	public void setEnterpriseNo(String enterpriseNo) {
		this.enterpriseNo = enterpriseNo;
	}
	public String getAccountNo() {
		return accountNo;
	}
	public void setAccountNo(String accountNo) {
		this.accountNo = accountNo;
	}
}

File: wms/src/main/webapp/system/app/model/cost/cost_AccountSetModel.js
/**
 * 模块名称：科目代码维护
 * 模块编码：B303
 * 创建：hcx 
 */
Ext.define('cms.model.cost.cost_AccountSetModel', {
    extend: 'Ext.data.Model',
    idProperty:'enterpriseNo,',
    fields: [
        {name:'enterpriseNo'},
		{name:'accountNo'},
		{name:'accountName'},
		{name:'createFlag'},
		{name:'remark'},
		{name:'rgstName'},
		{name:'rgstDate'},
		{name:'updtName'},
		{name:'updtDate'}
    ]
});